### ISSUE TITLE
Questions about reproduction

### ISSUE THREAD

**Issue by shonnon-zxs** [zehao-wang/LAD#6]
Why is 50k iter set in the paper, but 20k iter set in the code. I have already run 20k iters, and the best result is at 11k iters, but it is about 3% lower than the paper result. I want to know if this is the correct phenomenon. I used the warm_up2 you provided.

**Comment by zehao-wang** [comment]
Total iteration is set just for tuning the model, we select the model before overfitting (almost 100% SR in train subset), the final model I uploaded is from 16k if I remember correctly. Have you tried the final model we released to first verify the dataset? This checkpoint can even slightly outperform the number we reported in the paper on our machine. And when we do the challenge with REVERIE v2 half years ago, the host also check the code and can reproduce the results on their machine.

If you found the dataset is correctly set, probably tuning a bit with the loss might be helpful or loading the warmup checkpoint and resume the warmup stage for some iterations.
